src/supertux/main.cpp

```diff
@@ -366,8 +366,10 @@ Main::launch_game()
 
   timelog("audio");
   SoundManager sound_manager;
-  sound_manager.enable_sound(g_config->sound_enabled);
-  sound_manager.enable_music(g_config->music_enabled);
+  sound_manager.enable_sound(g_config->sound_volume > 0 ? true : false);
+  sound_manager.enable_music(g_config->music_volume > 0 ? true : false);
```

Comment:
I thought it would be better to disable sound if volume is 0 so we don't waste resources playing sounds with 0 gain. According to https://www.openal.org/documentation/openal-1.1-specification.pdf page 32 a gain of 0 is silence.